truffle.js

```diff
@@ -2,6 +2,39 @@ require("babel-register");
 require("babel-polyfill");
 const TestRPC = require("ethereumjs-testrpc");
 
+/**
+  In order to deploy normally without this setup you will have to comment all the code from =====>
+*/
+const Web3 = require("web3");
+
+const ProviderEngine = require("web3-provider-engine");
+const LedgerWalletSubproviderFactory = require("ledger-wallet-provider");
+const Web3Subprovider = require("web3-provider-engine/subproviders/web3.js");
+const FilterSubprovider = require("web3-provider-engine/subproviders/filters.js");
+
+const providerUrl = "http://localhost:8545";
+const nanoPath = "44'/60'/0'/0`";
+
+const web3HttpProvider = new Web3.providers.HttpProvider(providerUrl);
+const engine = new ProviderEngine();
+
+engine.addProvider(new FilterSubprovider());
+engine.addProvider(
+  LedgerWalletSubproviderFactory.default(new Web3(web3HttpProvider), nanoPath)
+);
+engine.addProvider(new Web3Subprovider(web3HttpProvider));
+engine.start();
+/**
+  =====> Till here
+  BUG: truffle cannot finish deployment without commenting the above section
```

Comment:
this is because of maybe too short timeout on Commitment contract START date... it is not important with automatic deployment